code/datums/votes/restart_vote.dm

```diff
@@ -19,36 +19,24 @@
 
 	return FALSE
 
-/datum/vote/restart_vote/toggle_votable(mob/toggler)
-	if(!toggler)
-		CRASH("[type] wasn't passed a \"toggler\" mob to toggle_votable.")
-
-	if(!check_rights_for(toggler.client, R_ADMIN))
-		return FALSE
-
+/datum/vote/restart_vote/toggle_votable()
 	CONFIG_SET(flag/allow_vote_restart, !CONFIG_GET(flag/allow_vote_restart))
-	return TRUE
 
 /datum/vote/restart_vote/is_config_enabled()
 	return CONFIG_GET(flag/allow_vote_restart)
 
-/datum/vote/restart_vote/can_be_initiated(mob/by_who, forced)
```

Comment:
why remove the `can_be_initiated()` message here? it's still useful for the info about the restart not auto-running if an admin is online to be present in the TGUI tooltip for this vote option.

the tgui_alert is fine to keep in addition to this imo.